stories/demos/CommonFeatures.tsx

```diff
@@ -52,6 +56,8 @@ interface Row {
   available: boolean;
 }
 
+const colSpanRowIds: number[] = [2, 4, 6];
```

Comment:
Could be a `Set`

Reply:
I have added a new `Column Spanning` example